Tests/UnitTests/Networking/Responses/Fixtures/ProductsEntitlements.json

```diff
@@ -0,0 +1,23 @@
+{
+    "products": [
+        {
+            "id": "com.revenuecat.foo_1",
```

Comment:
maybe we should also change the endpoint response to be "identifier"? wdyt? I imagine with gzip compression enabled it wouldn't really make a difference in terms of data transfer

Reply:
I agree. Is this endpoint ready? This is just a placeholder until we have the real format from the backend.